### ISSUE TITLE
EnC: Multi-process Cltr+F5 and attach

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#55761]
1. Create a console app project:

```C#
class Program
{
    static void Render()
    {
        Console.Write('.');
    }

    static void Main()
    {
        while (true)
        {
            Thread.Sleep(100);
            Render();
        }
    }
}
```

2. Ctrl+F5 twice and F5 once spawning 3 ConsoleApp.exe processes
3. Change `Console.Write('.');` to `Console.Write(':');` and Apply Updates while the processes are running

Observe that the change took effect in both processes.

4. Rename method `Render`.

Observe that a single rude edit error is reported in the Error List.

**Comment by tmat** [comment]
Currently broken - the debugger creates both hot reload and enc sessions separately, both are reporting diagnostics.

![image](https://user-images.githubusercontent.com/41759/130264236-e80264f2-b1bf-4100-9663-df8e8657091c.png)
